frontend/micro-ui/web/src/Customisations/UICustomizations.js

```
@@ -184,31 +183,43 @@ export const UICustomizations = {
 
-      const attendanceRegisterName = data?.body?.inbox?.moduleSearchCriteria?.attendanceRegisterName?.trim();
-      if(attendanceRegisterName) data.body.inbox.moduleSearchCriteria.attendanceRegisterName = attendanceRegisterName
+      const attendanceRegisterName =
+        data?.body?.inbox?.moduleSearchCriteria?.attendanceRegisterName?.trim();
+      if (attendanceRegisterName)
+        data.body.inbox.moduleSearchCriteria.attendanceRegisterName =
+          attendanceRegisterName;
 
       // deleting them for now(assignee-> need clarity from pintu,ward-> static for now,not implemented BE side)
       const assignee = _.clone(data.body.inbox.moduleSearchCriteria.assignee);
       delete data.body.inbox.moduleSearchCriteria.assignee;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)